<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogBrowserWidgetSimple</class>
 <widget class="QWidget" name="LogBrowserWidgetSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Page:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pageSpin">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::ButtonSymbols::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numPagesLabel">
       <property name="text">
        <string>%1 pages</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="beginButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="PyLogBook.qrc">
         <normaloff>:/PyLogBook/Resources/first-page.png</normaloff>:/PyLogBook/Resources/first-page.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previousButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="PyLogBook.qrc">
         <normaloff>:/PyLogBook/Resources/previous-page.png</normaloff>:/PyLogBook/Resources/previous-page.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="PyLogBook.qrc">
         <normaloff>:/PyLogBook/Resources/next-page.png</normaloff>:/PyLogBook/Resources/next-page.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="PyLogBook.qrc">
         <normaloff>:/PyLogBook/Resources/last-page.png</normaloff>:/PyLogBook/Resources/last-page.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="PyLogBook.qrc"/>
 </resources>
 <connections/>
</ui>
